packages/census/scripts/modify-snarkjs-package.js

```diff
@@ -0,0 +1,19 @@
+const fs = require('fs');
+const path = require('path');
+
+let packagePath = path.dirname(require.resolve("snarkjs"));
+
+while(!fs.existsSync(packagePath + '/package.json')) {
+    packagePath = packagePath.split('/');
+    packagePath.pop();
+    if(packagePath.length === 0) return;
+    packagePath = packagePath.join('/')
+}
+packagePath += '/package.json'
```

Comment:
I would use `path.join(packagePath, "package.json")` since it automatically concatenates using the right delimiter